src/angularjs/src/app/analysis-jobs/import/analysis-jobs-import.controller.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @ngdoc controller
+ * @name pfb.analysisJobs.import.controller:AnalysisJobImportController
+ *
+ * @description
+ * Controller for importing analysis results for a job run elsewhere
+ *
+ */
+(function() {
+    'use strict';
+
+    /** @ngInject */
+    function AnalysisJobImportController($log, $state, $filter, toastr, Neighborhood, AnalysisJobImport) {
```

Comment:
A couple unused imports:
```suggestion
    function AnalysisJobImportController($state, toastr, Neighborhood, AnalysisJobImport) {
```